simulators/carla_simulator.py

```diff
@@ -1,4 +1,4 @@
-from dataclasses import dataclass
+from dataclasses import dataclass, field
```

Comment:
I think everything CARLA-related should go into one folder, such as `examples/carla`.